website/static/js/share/results.js

```diff
@@ -37,79 +38,155 @@ Results.controller = function(vm) {
     self.vm = vm;
     self.vm.resultsLoading = m.prop(false);
 
+    self.renderTitleBar = function(result) {
+        return [m(
+            'a[href=' + result.uris.canonicalUri + ']', ((result.title || '').length > 0) ? m.trust(result.title) : 'No title provided'),
```

Comment:
I've been operating under the unverified assumption that elasticsearch wouldn't give me back dangerous results. I missed the usage of `m.trust`, though - that's worrying.

If `m.trust` is necessary, I'd like a comment here explaining why.